app/uk/gov/hmrc/transitmovementsauditing/controllers/AuditController.scala

```diff
@@ -67,12 +68,9 @@ class AuditController @Inject() (
       }
   }
 
-  private def convertIfNecessary(request: Request[Source[ByteString, _]]): EitherT[Future, PresentationError, Source[ByteString, _]] =
-    // TODO: is this okay to check for an XML content type?
-    if (
-      request.contentType.exists(
-        contentType => contentType.contains("/xml")
-      )
-    ) conversionService.toJson(request.body).asPresentation
+  private def convertIfNecessary(auditType: AuditType, request: Request[Source[ByteString, _]])(implicit
+    hc: HeaderCarrier
+  ): EitherT[Future, PresentationError, Source[ByteString, _]] =
+    if (request.contentType.contains(MimeTypes.XML)) conversionService.toJson(auditType.messageType, request.body).asPresentation
```

Comment:
Should contains be equals?

Reply:
`contentType` is an `Option[String]`, so `contains` is the same as `.map(x => x == MimeTypes.XML).getOrElse(false)`